apps/dg/english.lproj/strings.js

```diff
@@ -629,6 +629,7 @@ SC.stringsFor('English', {
   'DG.Inspector.newAttribute': "New Attribute in %@...",  // "New Attribute in %@..."
   'DG.Inspector.randomizeAllAttributes': "Rerandomize All", // "Randomize Attributes"
   'DG.Inspector.exportCaseData': "Export Case Data...", // "Export Case Data..."
+  'DG.Inspector.exportPrimaryCaseData.toolTip': "Export data from this case", // "Export Case Data..."
```

Comment:
It's exporting more than just this case, so perhaps something like "Export data from this case table" or "Export cases as .csv".